### PR TITLE
[Main Course] - fund data

### PR DESCRIPTION
**Pull request by jhqian0909**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Post for fund data(not for posting)

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Realvincentyuan — APPROVED** [review]
我们结合这些数据搞一个自己的服务也挺哇塞的。

**Review by Realvincentyuan — APPROVED** [review]
他们自己有开放给开发者的api吧？

**Comment by jhqian0909** [comment]
> 他们自己有开放给开发者的api吧？

并没有找到:(

**Review by EchoTang04 — APPROVED** [review]
LGTM, thanks Jiahui!
想知道网站有没有反爬机制，访问次数多了之后会不会有限制？

**Comment by EchoTang04** [comment]
> > 他们自己有开放给开发者的api吧？
> 
> 并没有找到:(

我找了一下，感觉很多都是非官方的第三方集成的API。好处是包含了多个网站的数据源，坏处是信息被采集和并不长期稳定靠谱。

File: docs/Main_Course/One_Day_Build/2021-11-27-fund_data.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+```
+其中，url就是我们刚刚找到数据对应的url。我们已经把数据整理成DataFrame，接着我们可以很方便地将数据导出，也可以作图等进一步分析。
+
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/img/fund5.png" width="500" />
+</figure>
+
+
+## 5 总结
+
```
Comment (Tina0330) [inline docs/Main_Course/One_Day_Build/2021-11-27-fund_data.md:73]: 谢谢佳辉！
总结是不是没有写完，还是就这样？

**Review by Tina0330 — APPROVED** [review]

File: docs/Main_Course/One_Day_Build/2021-11-27-fund_data.md (hunk 1 above)
Comment (jhqian0909) [reply docs/Main_Course/One_Day_Build/2021-11-27-fund_data.md:73]: 如果不发的话 就准备这样了。

**Comment by jhqian0909** [comment]
> LGTM, thanks Jiahui! 想知道网站有没有反爬机制，访问次数多了之后会不会有限制？

并没有尝试过访问太多次诶

**Review by jz263 — APPROVED** [review]

**Pull request merged**